src/mudlet.cpp

```diff
@@ -527,6 +527,10 @@ mudlet::mudlet()
             return;
         }
 
+        if (!host->mpEditorDialog && !createMudletEditor()) {
+            qWarning() << "Failed to create editor dialog";
```

Comment:
:thinking: Apropos my last comment about not needed to create the editor in this context it can actually be achieved now with the following:
```suggestion
        if (!host->mpEditorDialog) {
            qWarning() << "Failed to find editor dialog";
```

Reply:
I might be missing something, but wouldn't that render this PR ineffective?

Reply:
Nope, this is the lambda function that shows the editor and then the errors console within it if the menu item is clicked - but there is no point in creating the editor here as it must already have been done by the `mudlet` class method that creates the `Host` instance - remember we have reverted to creating the editor with the profile.

I suppose theoretically there might be a brief instant during the closing of a profile where the editor might not exist any more - but then trying to recreated the editor at that point would be a dumb thing to attempt! :boom:

Reply:
I think we are getting stuck in superficial details here while folks are crashing and are unable to launch Mudlet entirely. I'm having to recommend them to revert to an older installation for a couple of days now.

I don't think polishing the code to perfection and getting folks to re-test it again is the right thing to be doing. There's plenty of other imperfect code and the testers have already went through enough pain.